apps/dashboard/src/app/(dashboard)/(chain)/[chain_id]/[contractAddress]/modules/components/batchMetadata.stories.tsx

```diff
@@ -1,12 +1,12 @@
 import { Checkbox } from "@/components/ui/checkbox";
-import { CustomConnectWallet } from "@3rdweb-sdk/react/components/connect-wallet";
+import { getThirdwebClient } from "@/constants/thirdweb.server";
```

Comment:
server client, really?

Reply:
can be used on client side too - `useThirdwebClient` already does